src/index.ts

```diff
@@ -1,12 +1,16 @@
 import { useState, useEffect } from 'react';
 import { RefTaggerSettings } from './interfaces';
 
-const addScript = (setScriptAdded: React.Dispatch<React.SetStateAction<boolean>>): void => {
+const addScript = (
+  setScriptAdded: React.Dispatch<React.SetStateAction<boolean>>,
+  lang: string
+): void => {
   setScriptAdded(true);
+  lang = lang ? `.${lang}` : null;
```

Comment:
This will result in a bad URL below because
```
`${null}` === 'null'
```

We also prefer not reassign arguments, especially if the new value is semantically different.

```suggestion
  const languageSuffix = lang ? `.${lang}` : '';
```